test/NinjaTools.FluentMockServer.API.Tests.Exclude/Infrastructure/ScopedHttpContextRepository.cs

```diff
@@ -137,28 +137,25 @@ private void AddIsCalledWithKey<T>([NotNull] string key, [NotNull] T value)
 
 public class ExampleDataAttribute : Attribute
 {
-    [CanBeNull] 
-    public Type ExampleData { get; set; }
+    [CanBeNull] public Type ExampleData { get; set; }
 }
 
 public static class ExampleExtensions
 {
     public static Story Run<TStory>([NotNull] this object testObject, [NotNull] string scenarioTitle) where TStory : class
     {
-      //  var context = TestContext.GetContext(testObject);
+        //  var context = TestContext.GetContext(testObject);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4739715)